src/pudl/extract/eia_bulk_elec.py

```diff
@@ -24,7 +24,7 @@
 # thousands of times, clogging up the logs. Just warn us once!
 warnings.filterwarnings(
     action="once",
-    message="Could not infer format, so each element will be parsed individually, falling back to `dateutil`.",
+    message="Could not infer format, so each element will be parsed individually",
     category=UserWarning,
     module="pudl.extract.eia_bulk_elec",
 )
```

Comment:
This is extremely not working. Still getting tens-of-thousands of warnings when I run the ETL. What am I not understanding about how to use [`warnings.filterwarnings()`](https://docs.python.org/3/library/warnings.html#warnings.filterwarnings)